utils/fs/directory.go

```diff
@@ -46,6 +48,18 @@ func LastModified(search *LastModifiedSearch) (time.Time, error) {
 			}
 			return err
 		}
+		// Append the cwd if the paths are relative. This is needed because
+		// source/artifact handling (`filepath.Rel()`) needs absolute paths to work properly.
+		cwd, err := execenv.ValueFromFilesystem("cwd", "")
+		if err != nil {
+			return err
+		}
```

Comment:
This will always call `os.Getwd()`, right? I think we can call `os.Getwd()` directly here instead of exporting `ValueFromFilesystem`.

Reply:
You are absolutely right. Using `os.Getwd()` directly.